gateway/1.4/modules/ROOT/pages/_partials/task-tls-config.adoc

```diff
@@ -395,4 +395,83 @@ Flex Gateway supports multiple TLS certificates on a single gateway, but the cer
 ====
 Policy ordering cannot be applied to the Transport Layer Security policies.
 ====
-//end::config-ref[]
\ No newline at end of file
+//end::config-ref[]
+
+//tag::sample-config-tls-outbound-insecure-intro[]
+*Sample configuration for adding outbound TLS with no certificate validation:*
+//end::sample-config-tls-outbound-insecure-intro[]
+//tag::sample-config-tls-outbound-insecure[]
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: PolicyBinding
+metadata:
+  name: tls-outbound-insecure
+spec:
+  targetRef:
+    name: ingress-test-outbound-upstream
+    kind: Service
+  policyRef:
+    name: tls-outbound
+  config:
+    skipValidation: true
+----
+//end::sample-config-tls-outbound-insecure[]
+
+//tag::sample-config-tls-outbound-oneway-intro[]
+*Sample configuration for one-way outbound TLS:*
+//end::sample-config-tls-outbound-oneway-intro[]
+//tag::sample-config-tls-outbound-oneway[]
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: PolicyBinding
+metadata:
+  name: tls-outbound
+spec:
+  targetRef:
+    name: ingress-test-outbound-upstream
+    kind: Service
+  policyRef:
+    name: tls-outbound  
+  config:
+    trustedCA: |
+      # -----BEGIN CERTIFICATE-----
+      # insert certificate
+      # -----END CERTIFICATE-----
+----
+//end::sample-config-tls-outbound-oneway[]
+
+//tag::sample-config-mtls-intro[]
+*Sample configuration for one-way outbound mTLS:*
```

Comment:
"one-way outbound" and mTLS are mutually exclusive. In fact, mTLS and "two-way TLS" are interchangeable keywords. Should probably just be "Sample configuration for mTLS"